vendor/github.com/coreos/etcd/version/version.go

```diff
@@ -29,7 +29,7 @@ import (
 var (
 	// MinClusterVersion is the min cluster version this etcd binary is compatible with.
 	MinClusterVersion = "2.3.0"
-	Version           = "3.0.13"
+	Version           = "3.0.14"
```

Comment:
Heh - it seems that the change in Godeps is pretty much unneded, since we don't vendor etcdctl where all changes were...